controllers/storagecluster/cephcluster.go

```diff
@@ -714,6 +709,21 @@ func generateMonSpec(sc *ocsv1.StorageCluster, nodeCount int) cephv1.MonSpec {
 	}
 }
 
+func generateMgrSpec(sc *ocsv1.StorageCluster) cephv1.MgrSpec {
+	spec := cephv1.MgrSpec{
+		Modules: []cephv1.Module{
+			{Name: "pg_autoscaler", Enabled: true},
+			{Name: "balancer", Enabled: true},
+		},
+	}
+
+	if arbiterEnabled(sc) {
+		spec.Count = 2
```

Comment:
In addition to setting the spec.Count to 2, we would also need to set the resource limits/requests for the `resources.mgr-sidecar`. See [this commit](https://github.com/openshift/rook/pull/217/commits/9f03a0203089a703078443e102f4f6ed64f11746) for recommendations on the settings.